### PR TITLE
Linux fix for Silk.NET Vulkan Triangle experiment.

### PR DESCRIPTION
**Pull request by usertoroot**
# Summary of the PR
The validation layers cause a crash on my computer, proposed implementation more similar to https://github.com/KhronosGroup/Vulkan-Samples/blob/96dc1b3d729a743b7bb1caf5b9a631a5e684538b/samples/api/hpp_hello_triangle/hpp_hello_triangle.cpp#L135

# Related issues, Discord discussions, or proposals
https://github.com/dotnet/Silk.NET/pull/587

# Further Comments

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/Silk.NET?pullRequest=588) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/Silk.NET?pullRequest=588) before we can accept your contribution.<br/><br/>:x: usertoroot [sign now](https://cla.dotnetfoundation.org/dotnet/Silk.NET?pullRequest=588)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/Silk.NET?pullRequest=588) it.</sub>

File: src/Lab/Experiments/VulkanTriangle/HelloTriangleApplication.cs [hunk 1]
```diff
@@ -71,7 +71,20 @@ public void Run()
...
+            new [] { "VK_LAYER_KHRONOS_validation" },
+            new [] { "VK_LAYER_LUNARG_standard_validation" },
+            new [] 
+            {  
+	            "VK_LAYER_GOOGLE_threading",
+	            "VK_LAYER_LUNARG_parameter_validation",
+	            "VK_LAYER_LUNARG_object_tracker",
+	            "VK_LAYER_LUNARG_core_validation",
+	            "VK_LAYER_GOOGLE_unique_objects",
+            }
```
Comment (Perksey) [inline src/Lab/Experiments/VulkanTriangle/HelloTriangleApplication.cs:85]: Removing tabs

```suggestion
            new [] 
            {
                "VK_LAYER_GOOGLE_threading",
                "VK_LAYER_LUNARG_parameter_validation",
                "VK_LAYER_LUNARG_object_tracker",
                "VK_LAYER_LUNARG_core_validation",
                "VK_LAYER_GOOGLE_unique_objects",
            }
```

File: src/Lab/Experiments/VulkanTriangle/HelloTriangleApplication.cs [hunk 2]
```diff
@@ -257,13 +270,40 @@ private unsafe void CleanupSwapchain()
...
         private unsafe void CreateInstance()
         {
             _vk = Vk.GetApi();
 
-            if (EnableValidationLayers && !CheckValidationLayerSupport())
+            if (EnableValidationLayers)
             {
-                throw new NotSupportedException("Validation layers requested, but not available!");
+                _validationLayers = GetOptimalValidationLayers();
+                if (_validationLayers == null)
```
Comment (Perksey) [inline src/Lab/Experiments/VulkanTriangle/HelloTriangleApplication.cs:303]: nit: `is null` style

```suggestion
                if (_validationLayers is null)
```

File: src/Lab/Experiments/VulkanTriangle/HelloTriangleApplication.cs [hunk 3]
```diff
@@ -257,13 +270,40 @@ private unsafe void CleanupSwapchain()
             _vkSwapchain.DestroySwapchain(_device, _swapchain, null);
         }
 
+        private unsafe string[] GetOptimalValidationLayers()
```
Comment (Perksey) [inline src/Lab/Experiments/VulkanTriangle/HelloTriangleApplication.cs:273]: nit: nullables (don't think it's enabled in this project but doesn't hurt to annotate anyway)

```suggestion
        private unsafe string[]? GetOptimalValidationLayers()
```

**Review by Perksey — CHANGES_REQUESTED** [review]
Just a few nitpicks, happy to merge after these one-click fixes.

**Comment by usertoroot** [comment]
@Perksey Great! 

I'll use this opportunity to thank you and the team for making the library of my dreams supported by the .NET foundation.

**Review by Perksey — APPROVED** [review]
Thank you so much! 🥰

**Pull request merged**